src/kt_function.cpp

```diff
@@ -1,88 +1,95 @@
-#include <utility>

```

Comment:
I'm not sure what happened to the diff here. Line ending weirdness? I only made changes to `Variant KtFunction::invoke(const KtObject* instance, const Variant** p_args, int args_count)`